zzzeeksphinx/themes/zsbase/layout.mako

```diff
@@ -1,7 +1,14 @@
 ## coding: utf-8
 
 <%!
-    import datetime
+    import os
+    import time
+    from datetime import datetime, timezone
+
+    generated_at = datetime.fromtimestamp(
+        timestamp=float(os.environ.get('SOURCE_DATE_EPOCH', time.time())),
+        tz=timezone.utc if 'SOURCE_DATE_EPOCH' in os.environ else None,
```

Comment:
this would return a naive datetime and the `%Z` would fail.   we want a tz-aware datetime in all cases

Reply:
`%Z` returns an empty string for naive datetimes; my intent was to remain closer to the `datetime.now` behaviour when the env var is not set.

(and to do that, I read about the behaviour of [`datetime.now(None)`](https://docs.python.org/3.11/library/datetime.html#datetime.datetime.now)... and considered changing the fallback to `time.localtime()` (local naive time, I thought), but then read that [`datetime.today()`](https://docs.python.org/3.11/library/datetime.html#datetime.datetime.today) is the same as `datetime.fromtimestamp(time.time())`, so stayed with that)

Reply:
I think this should be a TZ aware timestamp and the %Z token should render in all cases